src/lib/Stage.class.ts

```diff
@@ -60,6 +60,8 @@ class Stage extends EventEmitter {
     this.generator.on('data', quad => {
       writer.addQuad(quad)
       quadCount ++
+      // slows down run test in Stage.class.test.ts
+      // this.emit('generatorResult', quadCount)
```

Comment:
find that it slows down a test, but I need this...

Reply:
I added this line `this.emit('generatorResult', quadCount)`, since I noticed it was missing in the stage class. I commented it out when noticing it would increase the timeout time for a test however, but this is not a problem.

- [ ] undo the comment and adjust the test accordingly